src/resources/nfts/simplehash/index.ts

```diff
@@ -113,3 +114,48 @@ export async function fetchSimpleHashNFTListing(
   );
   return cheapestListing;
 }
+
+/**
+ * Given an NFT, refresh its contract metadata on SimpleHash. If we can't,
+ * refresh metadata for this NFT only.
+ * @param nft
+ */
+export async function refreshNFTContractMetadata(nft: UniqueAsset) {
+  const chain = nft.isPoap
+    ? SimpleHashChain.Gnosis
+    : getSimpleHashChainFromNetwork(nft.network);
+
+  if (!chain) {
+    throw new Error(
+      `refreshNFTContractMetadata: no SimpleHash chain for network: ${nft.network}`
+    );
+  }
+
+  try {
+    await nftApi.post(
+      `/nfts/refresh/${chain}/${nft.asset_contract.address}`,
+      {},
+      {
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json',
+          'x-api-key': NFT_API_KEY,
+        },
+      }
+    );
+  } catch {
+    // If the collection has > 20k NFTs, the above request will fail.
+    // In that case, we need to refresh the given NFT individually.
+    await nftApi.post(
+      `/nfts/refresh/${chain}/${nft.asset_contract.address}/${nft.id}`,
```

Comment:
is this hitting SH directly or our own endpoint?

Reply:
our own, see: https://github.com/rainbow-me/rainbow/blob/@benisgold/refreshnft/src/resources/nfts/simplehash/index.ts#L15-L17